### ISSUE TITLE
[a11y] Navigation between internal hyperlinks

### ISSUE THREAD

**Issue by gregoriopellegrino** [readium/r2-testapp-swift#106]
When click on a link in the text the focus doesn't change, so VoiceOver remains to the origin content.

Tested on iOS 11.3 with VoiceOver

**Comment by mmenu-mantano** [comment]
Internal links were broken, not only with VoiceOver. It should be fixed by now in `develop`.

**Comment by mmenu-mantano** [comment]
Actually I misinterpreted this issue. The link is followed properly but the VO focus stays to the previous place until the user tap on the screen.
We might need to emit the notification https://developer.apple.com/documentation/uikit/uiaccessibilityscreenchangednotification?language=objc when switching to a different location.

**Comment by HadrienGardeur** [comment]
Issue moved to [readium/r2-navigator-swift #99](https://github.com/readium/r2-navigator-swift/issues/99) via [**ZenHub**](https://www.zenhub.com/)
